src/api/__tests__/Feed.test.js

```diff
@@ -532,6 +532,30 @@ describe('api/Feed', () => {
                 done();
             });
         });
+
+        test('should not include versions in feed items if shouldShowVersions is false', done => {
+            feed.feedItems(file, false, successCb, errorCb, errorCb, { shouldShowVersions: false });
+            setImmediate(() => {
+                expect(feed.versionsAPI.addCurrentVersion).not.toHaveBeenCalled();
```

Comment:
We've been trying to move away from using `toHave*` matchers and instead using `toBe*` matchers